tests/functional/scenario1/DatadirScenarioTest.php

```diff
@@ -442,6 +442,31 @@ public function testUser2HaveCorrectAccessAfterProvisioning(): void
         $this->assertCount(2, $user2TableRows);
     }
 
+    public function testQueryTagging(): void
```

Comment:
Tohle testuje něco jiného IMHO - to jestli se správně tagují queries v testech. Musel bych zavolat `$this->runAppWithConfig(self::getSchema1Config());`

Reply:
A protože to trvá věčnost, tak bych to runid nastavil a zkontroloval v některém z existujících testů...